export.go

```diff
@@ -7,7 +7,6 @@ type ExecResponse = execResponse
 type ExecResponseRowType = execResponseRowType
 type ExecResponseChunk = execResponseChunk
 type SnowflakeRows = snowflakeRows
-type SnowflakeChunkDownloader = snowflakeChunkDownloader
```

Comment:
This is actually a bug fix for our previous code.

This change on the left is introduced in our fork in https://github.com/observeinc/gosnowflake/commit/bf58bda332954a5fa303312584d57e006d394421

However, when we merged to 1.3.5, we dropped these code (silently): https://github.com/observeinc/gosnowflake/commit/be8eb31f3a5fce4dea58f1894752e1c18dc2d5c1#diff-9d9ca52939a19f26358e2b5b35ca938679b381be35f4c54379df94c4fa4d1233

Dropping makes sense, because I have checked that our monorepo is no longer using this export.

However, the dropping was silent and incomplete. It dropped MOST of the changes in bf58bda3, but not all. This line was missed. So let us drop all now.